### PR TITLE
feature/MBA-21_no_explicit_any_eslint

### PR DESCRIPTION
**Pull request by milkanuch**

### CODE REVIEW

File: src/components/TabModeSelector/TabModeSelectorItem/tabModeSelectorItem.types.ts [hunk 1]
```diff
@@ -1,10 +1,12 @@
+import { SignInMode } from '../../../screens/SignInScreen/signInScreen.types';
+
 export interface ITabModeSelectorItemProps {
-  currentValue: any;
-  changeValue: (newValue: string | number) => void;
+  currentValue: SignInMode;
```
Comment (stepan103ua) [inline src/components/TabModeSelector/TabModeSelectorItem/tabModeSelectorItem.types.ts:4]: Here should not be SignInMode, use instead string

File: src/components/TabModeSelector/tabModeSelector.types.ts [hunk 2]
```diff
@@ -1,10 +1,12 @@
 import { ViewStyle } from 'react-native';
 
+import { SignInMode } from '../../screens/SignInScreen/signInScreen.types';
+
 import { ITabModeSelectorItemData } from './TabModeSelectorItem/tabModeSelectorItem.types';
 
 export interface ITabModeSelectorProps {
-  currentValue: any;
-  setNewValue: (signInMode: any) => void;
+  currentValue: SignInMode;
```
Comment (stepan103ua) [inline src/components/TabModeSelector/tabModeSelector.types.ts:8]: Here should not be SignInMode, use instead string

**Review by stepan103ua — COMMENTED** [review]
Component TabModeSelector must be universal. So change SignInMode to string.

**Review by Xaiodo — APPROVED** [review]
LGTM

**Review by stepan103ua — APPROVED** [review]

**Pull request merged**